packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -416,6 +502,59 @@ public void resumeVideoRecording(@NonNull final Result result) {
     result.success(null);
   }
 
+  public void setFlashMode(@NonNull final Result result, String modeStr)
+      throws CameraAccessException {
+    // Get the flash availability
+    Boolean flashAvailable;
+    try {
+      flashAvailable =
+          cameraManager
+              .getCameraCharacteristics(cameraDevice.getId())
+              .get(CameraCharacteristics.FLASH_INFO_AVAILABLE);
+      if (flashAvailable == null || !flashAvailable) flashAvailable = false;
+    } catch (CameraAccessException e) {
+      result.error("setFlashModeFailed", e.getMessage(), null);
+      return;
+    }
+    // Check if flash is available.
+    if (!flashAvailable) {
```

Comment:
If we add a `null` check to this `if` statement, we can remove the `if` statement on line 514:

```suggestion
    if (flashAvailable == null || !flashAvailable) {
```

Reply:
Fixed